.gitignore

```diff
@@ -22,3 +22,5 @@ TabMonServiceInstaller/Generated/
 *.user
 docs/Gemfile.lock
 docs/_site
+TabMonConfigBuilder/bin/
```

Comment:
nit: We could think about using a wildcard for bin dirs.

Reply:
That's a good point.